controllers/heatapi_controller.go

```diff
@@ -329,6 +329,8 @@ func (r *HeatAPIReconciler) reconcileInit(
 		return ctrlResult, nil
 	}
 
+	instance.Status.Conditions.MarkTrue(condition.ExposeServiceReadyCondition, condition.InputReadyMessage)
```

Comment:
Is the `condition.InputReadyMessage` right?  Should it maybe be `condition.ExposeServiceReadyMessage`?

Reply:
Ohh, I copied it from heatcfnapi but that contains the same problem. I'll fix it soon.

Reply:
Ignore my comment above. The implementation in cfnapi looks correct. I've submitted https://github.com/openstack-k8s-operators/heat-operator/pull/173/files .